superset-frontend/src/profile/components/App.tsx

```diff
@@ -32,6 +31,14 @@ interface AppProps {
   user: UserWithPermissionsAndRoles;
 }
 
+const StyledPanel = styled.div`
+  padding: 10px;
```

Comment:
This padding adds space around the entirety of the content area... this wasn't in the original layout, and probably isn't needed in this one. It makes the gutters pretty large. I would just take this out.

![Uploading padding.gif…]()

Reply:
Also note that the screenshot used in the PR description doesn't show the Panels being removed by the PR.

Reply:
Oops wrong screenshot. Will adjust the padding.